taccsite_cms/templates/guides/getting_started.html

```diff
@@ -309,7 +332,7 @@ <h4 id="setup-mfa">Multi-Factor Authentication</h4>
       <h4 id="request-access">Requesting Access</h4>
       <p>
         To use the Portal, you may need to have an active allocation on
-        TACC resources. If you are PI eligible, you may request your own allocations. Alternatively, you may request to be added to an
+        TACC resources. If you are PI eligible, you may request your own allocations. Alternatively (if you do not already have an active allocation), you may request to be added to an
         allocation through the
         <a href="{{ site_uri.absolute_uri }}workbench/onboarding/setup" target="_blank">Account Onboarding</a>.
```

Comment:
I think it would read better this way:

"..., you may request access to an existing allocation during the Account Onboarding process."

Reply:
Text is now updated to match your suggestion (8dc64ed). Thank you.